spec/requests/api/v1/drivers_spec.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   it 'responds with drivers json' do
-    get '/api/v1/drivers'
+    response = doorkeeper_token.get('/api/v1/drivers')
```

Comment:
I would like to discuss the request tests, I would like to see the following (if possible)...

1. Contexts for valid, invalid, and no application details
  - when application id and secret are valid
  - when application id and secret are not valid
  - when application id and secret are not present
2. Some cleaner helper that overrides, extends, or an addition to get
  - doorkeeper_get for instance